example/package.json

```diff
@@ -23,6 +23,7 @@
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
     "webpack": "^4.39.2",
+    "webpack-cli": "^3.3.10",
```

Comment:
What is this needed for?

Reply:
The example was missing this dependency. It was going one folder up to the root folder to get it before. Once I removed it from the root folder (as a cleanup step to completely removing Webpack from the build process), the example no longer worked.

Reply:
Ahhh good catch.